README.md

```diff
@@ -193,55 +172,51 @@ Define C++ flags for the platform:
-5. Main UI loop with `webview_loop()` inside has been replaced with `webview_run()` runs infinitely until the webview window is closed.
-6. `webview_terminate()` remains the same.
-7. `webview_dispatch()` remains the same.
-8. `webview_set_title()` remains the same.
-9. `webview_set_color()` has been removed. Use `webview_get_window` and native
-   window APIs to control colors, transparency and other native window
-   properties. At some point these APIs might be brought back.
-10. `webview_set_fullscreen()` has been removed, see above.
-11. `webview_dialog()` has been removed. But I'd like to see it added back as a separate independent module or library.
-12. `webview_eval()` remains the same.
-13. `webview_inject_css()` has been removed. Use `webview_eval()` to create style tag manually.
-14. `webview_debug()` has been removed. Use whatever fits best to your programming language and environment to debug your GUI apps.
+Full C/C++ API is described at the top of `webview.h` and at https://webview.dev.
 
 ## Notes
 
-- A webview is not a full web browser and thus does not support `alert`, `confirm` and `prompt` dialogs. Additionally, `console.*` methods are not supported.
+- A webview is not a full web browser. Although they may work, we do not support `alert`, `confirm` and `prompt` dialogs. Additionally, `console.*` methods are not supported.
 - Ubuntu users need to install the `webkit2gtk-4.0` as development dependency via `sudo apt install webkit2gtk-4.0`. If the package can't be found `webkit2gtk-4.0-dev` may be used instead.
 - FreeBSD is also supported via webkit2 which may be installed by running `pkg install webkit2-gtk3`.
 - Execution on OpenBSD requires `wxallowed` [mount(8)](https://man.openbsd.org/mount.8) option.
-- Calling `Eval()` or `Dispatch()` before `Run()` does not work, because the webview instance has only been configured, but not started yet. 
+- On Windows, users must install:
+  - Windows 10 SDK via Visual Studio Installer
+  - C++ support via Visual Studio Installer
+  - Install webview2 (you may already have this) [download from Microsoft](https://developer.microsoft.com/en-us/microsoft-edge/webview2/)
+- Calling `Eval()` or `Dispatch()` before `Run()` does not work, because the webview instance has only configured, but not started yet. 
+
+## Contributing
+Contributing to the webview project is always welcome! We are especially in need of MacOs developers. Active maintainers who review pull requests and triage issues are listed here.
+- @justjosias
+- @dandeto
+- @nicklasfrahm
+
+### Guidelines for Contributing Examples
+All examples will be held to the same standard as the main codebase.
+
+Additionally, examples should...
+- Be cross platform except under certain circumstances
+- Highlight a subset of the webview library's API
+- Be well documented
+- Have a simple goal in mind and lack large dependencies
+- Link to external libraries instead of copying them to the webview repo
```

Comment:
Great idea! Maybe this should be in a `CONTRIBUTING.md` file instead of the README? That seems to be standard, since it's not applicable to most users.